Makefile

```diff
@@ -352,23 +393,26 @@ add-on: node_modules \
 	      $(BUILT)/$(ADD-ON)/chrome.manifest \
 	      $(BUILT)/$(ADD-ON)/chrome/locale/chrome.manifest \
 	      $(BUILT)/$(ADD-ON)/install.rdf \
-	      $(BUILT)/add-on/chrome/content/preferences/prefs.js
+	      $(BUILT)/add-on/chrome/content/preferences/prefs.js \
+				$(BUILT)/add-on/bootstrap.js
```

Comment:
To be clear, this wrong indent doesn't matter with `bootstrap.js` as part of `built_add_on_js_files`.